af/oometa/006-test-lists.install.sql

```diff
@@ -0,0 +1,42 @@
+BEGIN;
+
+select _v.register_patch( '006-test-lists', ARRAY[ '005-repeated-report' ], NULL );
+
+CREATE SEQUENCE IF NOT EXISTS cat_no_seq;
+
+CREATE TABLE IF NOT EXISTS url_categories
+(
+    cat_no INT NOT NULL default nextval('cat_no_seq') PRIMARY KEY,
+    cat_code VARCHAR UNIQUE NOT NULL,
+    cat_desc VARCHAR NOT NULL,
+    cat_long_desc VARCHAR,
+    cat_old_codes VARCHAR
```

Comment:
I'm not 100% sure, but you probably want it represented as array of strings.

Reply:
Yes that makes sense. Will do that.